lib/src/command/token_add.dart

```diff
@@ -82,15 +80,15 @@ class TokenAddCommand extends PubCommand {
   }
 
   Future<void> _addEnvVarToken(Uri hostedUrl) async {
```

Comment:
```suggestion
  Future<void> _addEnvVarToken(Uri hostedUrl, String envVar) async {
```

Then we don't have to do: `envVar!` all over the place.